src/sentry/options/manager.py

```diff
@@ -29,6 +29,11 @@ class UnknownOption(KeyError):
 # config files.
 FLAG_STOREONLY = 0b100
 
+# How long will a cache key exist in local memory before being evicted
+DEFAULT_KEY_TTL = 10
```

Comment:
It does not. The network cache TTL is configured at the `OptionsStore.__init__`.

`cache.set` uses `self.ttl`, not `key.ttl`.